thunderbird/processes/wps_generate_prsn.py

```diff
@@ -149,9 +151,12 @@ def _handler(self, request, response):
                 self, response, "Dry Run", process_step="dry_run", level=loglevel
             )
             del response.outputs["output"]  # remove unnecessary output
-            dry_output_path = os.path.join(self.workdir, "dry.txt")
-            dry_run_info(filepaths, dry_output_path)
-            response.outputs["dry_output"].file = dry_output_path
+            dry_file = dry_run_info(
+                os.path.join(self.workdir, "dry.txt"),
```

Comment:
Could we use your new method here?

Reply:
@nikola-rados Actually, this is harder than it looks. `generate_prsn` has 3 inputs, thus the naming convention `input_filename_dry.txt` should not be applied in this case. Do you have any good idea how to deal with this situation?

Reply:
I'm not sure I understand what the issue is with the naming convention, are we not still creating a single dry output file?

Reply:
> are we not still creating a single dry output file?

Yes, and that is the problem we want to deal with. For example, when running `wps_generate_climos` dry_run on an imaginary file `non_climo.nc`. The output file will be named `non_climo_dry.txt`. On the other hand, assume running `wps_generate_prsn` dry_run on `pr.nc`, `tasmin.nc` and `tasmax.nc`, we cannot follow the output file convention.